Sample/SampleDelegate/SampleDelegate/AppDelegate.swift

```diff
@@ -33,65 +33,65 @@ class AppDelegate: SuperDelegate, ApplicationLaunched {
         // Setup app model, networking, logging, etc.
     }
     
-    func loadInterfaceWithLaunchItem(launchItem: LaunchItem) {
+    func loadInterface(launchItem: LaunchItem) {
         // Setup our main window since we don't have a storyboard.
-        setupMainWindow(window)
+        setup(mainWindow: window)
         window.rootViewController = ViewController()
         
         switch launchItem {
-        case let .RemoteNotificationItem(remoteNotification):
+        case let .remoteNotification(item):
             // Show UI for the remote notification that brought us into the foreground.
-            didReceiveRemoteNotification(remoteNotification, notificationOrigin: .UserTappedToBringAppToForeground, fetchCompletionHandler: { (_) in
+            didReceive(remoteNotification: item, origin: .userTappedToBringAppToForeground, fetchCompletionHandler: { (_) in
                 // Nothing to do here.
             })
             
-        case let .LocalNotificationItem(localNotification):
+        case let .localNotification(item):
             // Show UI for the local notification that brought us into the foreground.
-            didReceiveLocalNotification(localNotification, notificationOrigin: .UserTappedToBringAppToForeground)
+            didReceive(localNotification: item, origin: .userTappedToBringAppToForeground)
             
-        case let .OpenURLItem(urlToOpen):
+        case let .openURL(item):
             // If we have a urlToOpen launch item, SuperDelegate has guaranteed that we can open that URL.
-            handleURLToOpen(urlToOpen)
+            let _ = handle(urlToOpen: item)
```

Comment:
you can. I like not ignoring the fact that there's a value returned. This is the Swift equivalent of ObjC `(void)[self handleURLToOpen:url]`. I'm just calling out that I'm ignoring the return value.